dev/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM quay.io/astronomer/astro-runtime:12.2.0
+
+USER root
+
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 && \
+    chmod 700 get_helm.sh && \
+    ./get_helm.sh
+
+USER astro
```

Comment:
do we need this?

Reply:
just want to install whl in astro user space

Reply:
For some DAG we run helm command, so we need it https://github.com/astronomer/astro-provider-ray/blob/4a362a268011326db0520fa3caad1c97aafa2c05/ray_provider/hooks/ray.py#L594

Reply:
i see, thanks for clarifying.